releasenotes/notes/add-get-tapered-mapper-for-problems-33292452930141d1.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Adds the method 
+    :meth:`~qiskit_nature.second_q.problems.BaseProblem.get_tapered_mapper()` to
+    transform a :class:`~qiskit_nature.second_q.mappers.QubitMapper` instance into a 
+    :class:`~qiskit_nature.second_q.mappers.TaperedQubitMapper` based on the properties of the current
+    problem.
+  - |
+    Update API for the method
+    :meth:`~qiskit_nature.second_q.problems.BaseProblem.symmetry_sector_locator()`
+    to accept ``Z2Symmetries`` as defined in the quantum_info module as well as the ``Z2Symmetries``
+    defined in the opflow module.
```

Comment:
This also needs to mention the addition of the `TaperedQubitMapper` itself. Ideally, that should also show an example of how this is used (you can also reuse the `get_tapered_mapper` feature for that example.